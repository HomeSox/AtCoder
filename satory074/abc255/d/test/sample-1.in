5 5
6 11 2 5 5
5
20
0
7
3
