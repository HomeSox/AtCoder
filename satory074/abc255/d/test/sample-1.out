10
71
29
14
16
